trantor/net/EventLoop.h

```diff
@@ -297,9 +297,9 @@ class TRANTOR_EXPORT EventLoop : NonCopyable
     void abortNotInLoopThread();
     void wakeup();
     void wakeupRead();
-    bool looping_;
+    volatile bool looping_;
```

Comment:
Maybe std::atomic<bool> should be used here.

Reply:
@gschmottlach-xse Please use `std::atomic<bool>`. volatile does not imply atomic. Accessing a volatile variable from multiple threads is still a UB from the compiler's point of view.